### ISSUE TITLE
Marking Categories as Hidden does not work

### ISSUE THREAD

**Issue by Mad-Spy** [BattleScribe/Release#53]
Marking a category as hidden does not actually hide that category on the output. If this could be made to work so that we can use modifiers to hide/unhide categories based on Roster selections, that would be great.

**Comment by Jonskichov** [comment]
As of 2.02.03 on desktop, categories are now properly hidden in the left-hand catalogue tree. But that's not what you're asking for is it? :)

**Comment by Mad-Spy** [comment]
No. What I'd like is to be able to selectively hide Categories from being displayed on the output based on Roster selections.

This would allow us to switch "Faction" or "unit type" in catalogues without having to duplicate Root Selection Entries.
